packages/abi-gen/test-cli/output/typescript/abi_gen_dummy.ts

```diff
@@ -323,7 +329,7 @@ export class AbiGenDummyContract extends BaseContract {
         awaitTransactionSuccessAsync(
             wad: BigNumber,
             txData?: Partial<TxData>,
-            opts: SendTransactionOpts = { shouldValidate: true },
+            opts: AwaitTransactionSuccessOpts = { shouldValidate: true, timeoutMs: 0 },
```

Comment:
Defaulting `timeoutMs` to 0 doesn't make any sense. From the doc comment, this is: `How long (in ms) to poll for transaction mined until aborting.`. Aborting after 0 MS makes it impossible for the poll request to complete. Am I missing something?

Reply:
Ahh good catch. This should be `pollingIntervalMs` and was only set to 0 in contracts unit tests. `@0x/web3wrapper` has a default `pollingIntervalMs = 1000` already so I will leave the wrapper without a default value and keep the existing behaviour.